### PR TITLE
Move kiwi-images-wsl from openSUSE:Factory:WSL to openSUSE:Factory

### PR DESCRIPTION
**Pull request by Vogtinator**
It's built like the other parts now.

`scriptgen.py` didn't handle `folder` for assets properly, I had to fix that.

### CODE REVIEW

File: t/obs/openSUSE:Factory:ToTest/wsl/print_rsync_iso.before [hunk 1]
```diff
@@ -0,0 +1,2 @@
+rsync --timeout=3600 -tlp4 --specials obspublish::openqa/openSUSE:Factory:ToTest/appliances/x86_64/*openSUSE-Tumbleweed-20210701-WSL.x86_64-21153.2.2.0-Snapshot20210701.appx /var/lib/openqa/factory/other/openSUSE-Tumbleweed-20210701-WSL.x86_64-21153.2.2.0-Snapshot20210701.appx
```
Comment (andrii-suse) [inline t/obs/openSUSE:Factory:ToTest/wsl/print_rsync_iso.before:1]: kiwi-images-wsl is still not in the path, I will have a look how to fix it

**Review by andrii-suse — CHANGES_REQUESTED** [review]

File: t/obs/openSUSE:Factory:ToTest/wsl/print_rsync_iso.before (hunk 1 above)
Comment (andrii-suse) [reply t/obs/openSUSE:Factory:ToTest/wsl/print_rsync_iso.before:1]: I cannot push to your repo, so could you add this commit and all should be good 
https://github.com/andrii-suse/openqa-trigger-from-obs/commit/acd7da493

**Review by andrii-suse — APPROVED** [review]
All should be good once the comment is addressed

File: t/obs/openSUSE:Factory:ToTest/wsl/print_rsync_iso.before (hunk 1 above)
Comment (Vogtinator) [reply t/obs/openSUSE:Factory:ToTest/wsl/print_rsync_iso.before]: Done

**Pull request merged**